### PR TITLE
Plasmeme helmet fix

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request

Adds "replacement" plasmeme helmets for the clothing vendors, along with a few new plasmeme uniforms for the command staff, clown, and mime.

## Why It's Good For The Game

Since the PR that added plasmemes spaceproof, flashproof, helmet into almost every vendor in every department, with some departments getting about 4-6 of them total(Medical getting nearly 10) for free, people have been powergaming with roundstart buying them for their unrestricted flash protection, effectively defeating the reason why were put into the vendors in the first place; to make sure plasmeme players had a replacement helmet/suit when they were cloned, or when they switched jobs.  The replacement helmets are still spaceproof, but lack the flash protection of a standard helmet.

Additionally I found it fairly stupid that plasmemes have uniforms for every role other then command roles, so I made some to go with this PR, along with one for the clown and mime.

## Changelog
:cl:
add: Replacement plasmeme helmets which replace all the normal helmets in every department vendor.
add: Command, clown, and mime plasmeme helmets
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
alright travis, fuck me and tell me what I did wrong.

File: code/modules/clothing/under/jobs/Plasmaman/medsci.dm [hunk 1]
```diff
@@ -39,3 +39,18 @@
 	icon_state = "chemist_envirosuit"
 	item_state = "chemist_envirosuit"
 	item_color = "chemist_envirosuit"
+	
+/obj/item/clothing/under/plasmaman/cmo // i hope qwerty-chan appriciates this UwU
```
Comment (qwertyquerty) [inline code/modules/clothing/under/jobs/Plasmaman/medsci.dm]: Remove the comment.

Comment (ghost) [reply code/modules/clothing/under/jobs/Plasmaman/medsci.dm]: fiiiiiine

**Comment by qwertyquerty** [comment]
Fix your changelog

**Comment by ghost** [comment]
wow a new record, travis only told me to fuck off once.

**Review by qwertyquerty — APPROVED** [review]

**Comment by ghost** [comment]
<3

**Comment by ghost** [comment]
bump

**Comment by ike709** [comment]
Vote passed.

**Pull request merged**